core/src/main/java/com/hcl/ecomm/core/models/StaticListModel.java

```diff
@@ -0,0 +1,46 @@
+package com.hcl.ecomm.core.models;
+
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.models.annotations.Model;
+import org.apache.sling.models.annotations.Optional;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.PostConstruct;
+import javax.inject.Inject;
+import java.util.ArrayList;
+import java.util.List;
+@Model(adaptables = Resource.class)
+public class StaticListModel {
```

Comment:
@Rajamouli-hcl  why you need another custom model?

Reply:
Listmodel is used for getting the input values from the dialog and StaticListModel is used for getting the stored values from the resource by adapting  to the ListModel